src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss

```diff
@@ -0,0 +1,48 @@
+@import 'scss/fonts/_fonts';
+@import 'scss/variables/_colors';
+@import 'scss/variables/_content';
+
+.banner {
+  background-color: $mine-shaft;
+  min-height: 20vh;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: $max-content;
+  padding: 15px;
+  width: 95vw;
+}
+
+.title {
+  color: $white;
+  font-family: $source-sans-pro;
+  font-size: 45px;
+  font-weight: 600;
```

Comment:
Sort

Reply:
This one looks good to me, font-size goes before font-weight.